### PR TITLE
Add Flow to geography package

### PR DESCRIPTION
**Pull request by Floppy**
Done as a learning exercise (also cos we should).

### CODE REVIEW

**Comment by Floppy** [comment]
So I've learned a lot about babel, flow, and other things like that. I just can't work out the last flow error currently causing the failure.

File: README.md [hunk 1]
```diff
@@ -12,12 +12,23 @@ const {
...
+OR
+
+import {
+  regions,
+  countriesInRegion,
+  countryCodesInRegion,
+  countryNamesInRegion,
+} from '@apolitical/geography';
+
+
```
Comment (Gisleburt) [inline README.md]: Have you checked this using `npm link` with your existing libraries?

File: README.md [hunk 2]
```diff
@@ -12,12 +12,23 @@ const {
...
 regions() // Get a list of region codes
 countryNamesInRegion('afr') // Country names in Africa
 countryCodesInRegion('eur') // Country code in Europe
 countriesInRegion('na') // Code->name map for North America
 ```
 
+For IE support, you will need to use [`babel-polyfill`](https://github.com/babel/babel-preset-env#usebuiltins) in your application.
```
Comment (Gisleburt) [inline README.md]: Possibly not just IE. Perhaps just "full browser support"?

File: flow-typed/npm/mocha_v5.x.x.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+  declare export var before: typeof before;
+  declare export var after: typeof after;
+  declare export var after: typeof after;
+  declare export var beforeEach: typeof beforeEach;
+  declare export var beforeEach: typeof beforeEach;
+  declare export var afterEach: typeof afterEach;
+  declare export var afterEach: typeof afterEach;
+
+  declare export default $npm$mocha$Mocha;
+}
```
Comment (Gisleburt) [inline flow-typed/npm/mocha_v5.x.x.js]: Could this be imported from somewhere just in case it changes?

Comment (Floppy) [reply flow-typed/npm/mocha_v5.x.x.js]: flow-typed seems to want to import it into the repo - I've not seen a way to use it via a module, though there may be one. I'll take a look.

File: README.md (hunk 1 above)
Comment (Floppy) [reply README.md]: Have now!

**Pull request merged**